packages/storage-evm/scripts/base-mainnet-contracts-scripts/configure-ownership-token-contracts.ts

```
@@ -0,0 +1,245 @@
+
+  // Extract contract addresses using regex
+  const patterns = {
+    OwnershipTokenFactory:
+      /OwnershipTokenFactory proxy deployed to: (0x[a-fA-F0-9]{40})/,
+    OwnershipTokenVotingPowerImplementation:
+      /OwnershipTokenVotingPowerImplementation proxy deployed to: (0x[a-fA-F0-9]{40})/,
+  };
+
+  for (const [key, pattern] of Object.entries(patterns)) {
+    const match = fileContent.match(pattern);
+    if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)